### DISCUSSION TITLE
Just letting you know the rpi 3 a+ is g <s>too slow</s> fast enough to run birdnetpi .

### DISCUSSION THREAD

**Discussion by hopdebee** [mcguirepr89/BirdNET-Pi#382] [Show and tell]
I tried it, even while cooling it with two fans, https://i.imgur.com/Vmx3kg6.jpg (rpi is in the box under the two fans). It kept the temperature within 30-33 degrees Celcius. Without it, the unit got pretty hot. Alas, even with this cooling it cannot run it smoothly enough to be usable.

**Comment by mcguirepr89** [comment]
Hello! Thanks for the info! Did you add any swap space?

https://github.com/mcguirepr89/BirdNET-Pi/issues/231

https://github.com/mcguirepr89/BirdNET-Pi/discussions/242#discussioncomment-2913963

**Comment by hopdebee** [comment]
Ok interesting things I might try ! Thanks for pointing me in this direction. Will test and report.

**Comment by hopdebee** [comment]
With the settings for the zero, https://github.com/mcguirepr89/BirdNET-Pi/wiki/RPi0W2%20Installation%20Guide, I managed to get the birdnetpi running smooth enough on the rpi 3 a+ ! I'm still cooling the CPU with two full size computer fans (its 30 degrees Celcius today). This might not be necessary in cooler weather. I also disabled a number of services as instructed. Maybe that is not needed either. If i experiment with extra load/less cooling and find something significant i'll let you guys know here. Thanks for this nifty application. Impressive !
